fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java

```diff
@@ -195,9 +196,12 @@ private StatsDeriveResult computeFilter(Filter filter) {
     //       3. Get NDV and column data size from StatisticManger, StatisticManager doesn't support it now.
     private StatsDeriveResult computeScan(Scan scan) {
         Table table = scan.getTable();
-        TableStats tableStats = Utils.execWithReturnVal(() ->
-                ConnectContext.get().getEnv().getStatisticsManager().getStatistics().getTableStats(table.getId())
-        );
+        TableStats tableStats = Utils.execWithReturnVal(() -> {
+            Statistics statistics = ConnectContext.get().getEnv().getStatisticsManager().getStatistics();
+            // TODO: tmp mock the table stats, after we support the table stats, we should remove this mock.
+            statistics.mockTableStatsWithRowCount(scan.getTable().getId(), 0);
```

Comment:
u should use scan node to compute row count just like what original planner do rather than set row count to zero